emcc.py

```diff
@@ -3354,11 +3354,14 @@ def modularize():
 
   src = '''
 function(%(EXPORT_NAME)s) {
+  let resolve, reject;
```

Comment:
I like the look of this PR in general, good work!

Maybe give these a longer name so that it's clear when you come across them in other files what their purpose is? Maybe `exported_promise_resolve`?

Reply:
Another option might be to add a comment here, and keep the identifier names shorter..

Reply:
But yeah, maybe as these identifiers are used elsewhere, a longer name might be useful. `resolvePromise, rejectPromise` perhaps?

Reply:
Those suggestions sound good to me. Any feedback other than the name bikeshedding?